R/zzz.R

```diff
@@ -14,7 +17,7 @@ is_testing <- function() {
     extrafont::loadfonts("win", quiet = TRUE)
   }
 
-  fnt <- extrafont::fonttable()
```

Comment:
@maelle I made a mistake here, this line shouldn't have been changes

Reply:
ok, and once corrected does it pass?

Reply:
it builds, it's not tested though, as discussed above :)